### DISCUSSION TITLE
vue-cesium:如何避免这类的类型报错

### DISCUSSION THREAD

**Discussion by xiaowunimte** [zouyaoji/vue-cesium#618] [Q&A]
![1729558325868](https://github.com/user-attachments/assets/0a3f7fc6-e4aa-4574-b44a-47e394418b8d)
![1729558390853](https://github.com/user-attachments/assets/7ec3582c-31f6-45e0-a4be-53d0602a391e)

**Comment by zouyaoji** [comment]
在vc-viewer的ready事件之后再用cesium相关的API。
